### DISCUSSION TITLE
Is it possible to run this with the stress option from terminal?

### DISCUSSION THREAD

**Discussion by zeonos** [amanusk/s-tui#185] [Q&A]
I am trying to stress test a bunch of Raspberry Pi's that are mounted in different locations and my idea was to just have a SD card with stress and s-tui to run some stress test, it would be easiest if i could just auto start s-tui in the stress mode, so i dont need any input device.

Is there a command to allow me to open s-tui in stress mode?

**Comment by amanusk** [comment]
Currently no such option, but s-tui uses the default stress package, you could perhaps run them both and only use s-tui to collected data
